scenarioo-server/src/main/java/org/scenarioo/dao/search/dao/SearchResultsDao.java

```diff
@@ -0,0 +1,50 @@
+/* scenarioo-server
+ * Copyright (C) 2014, scenarioo.org Development Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.scenarioo.dao.search.dao;
+
+import java.util.Collections;
+import java.util.List;
+
+public class SearchResultsDao {
```

Comment:
I renamed the `Dao`s to `SearchableXyz`, as they are used for indexing and for the search results. Furthermore I renamed the MetaData to `SearchableObjectContext`. I chose this rather long name because it is also used to ignore this part in the index (see `ElasticSearchIndexer`) and I want to avoid that we ignore everything that has "Context" in the name in the index.